### PR TITLE
chore: baekjoon 2753

### PR DESCRIPTION
**Pull request by kyun-9458**
https://www.acmicpc.net/problem/2753

### CODE REVIEW

File: src/lightvirus/baekjoon/_2753.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        Scanner sc = new Scanner(System.in);
+        int a = sc.nextInt();
+
+        if (a % 4 == 0) {
+            if (a % 100 != 0 || a % 400 == 0) {
+                System.out.println("1");
+            } else {
+                System.out.println("0");
+            }
+        } else if (a % 4 != 0) {
```
Comment (zeroradish) [inline src/lightvirus/baekjoon/_2753.java:16]: unwrap if statement  :blush:

**Review by zeroradish — APPROVED** [review]
:+1:

**Review by joyowlsf — APPROVED** [review]
확인완료 !

**Review by Spidyweb-3588 — APPROVED** [review]
윤년 확인 완료했습니다!

**Pull request merged**